src/uikit.cs

```diff
@@ -4077,6 +4088,11 @@ public interface UIColor : NSSecureCoding, NSCopying {
 		[Export ("colorWithCGColor:")][Static]
 		UIColor FromCGColor (CGColor color);
 
+		[iOS (10,0)]
+		[Static]
+		[Export ("colorWithDisplayP3Red:green:blue:alpha:")]
+		UIColor FromDisplayP3Red (nfloat displayP3Red, nfloat green, nfloat blue, nfloat alpha);
```

Comment:
In C#, the argument names are not generally visible, so the P3 needs to be visible in the method name to express the intent.